.eslintrc

```diff
@@ -1,49 +1,40 @@
 {
   "extends": "airbnb",
```

Comment:
Yes I thought of that myself. I just didn't want to change everything ;) Let's give it a try.